### PR TITLE
[dependencies] Bump cheerio to v1.0.0-rc10

### PR DESCRIPTION
**Pull request by verbose117**
### Related Issues

> Fixes None
> Related to None

#### Tests

- [x] The code changed/added as part of this PR has been covered with tests
- [x] All tests pass locally with `yarn test-unit`

#### Code Review

- [x] Issue from task tracker has a link to this PR
- [x] This PR has a concise title and thorough description useful to a reviewer

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "@typescript-eslint/parser": "4.28.0",
     "async-retry": "1.2.3",
     "buffer-replace": "1.0.0",
-    "cheerio": "1.0.0-rc.3",
+    "cheerio": "v1.0.0-rc.10",
```
Comment (styfle) [inline package.json]: This also needs to update the `yarn.lock` file.

You can do that by running `yarn install`

Comment (verbose117) [reply package.json]: Done ☑️

Comment (styfle) [reply package.json]: I think the `v` prefix should be removed too 

https://www.npmjs.com/package/cheerio

image

Comment (verbose117) [reply package.json]: I have removed it 👍🏻

**Pull request closed without merging**